plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstance.java

```diff
@@ -246,6 +246,11 @@ private String generateRepository() {
 
     @Override
     public void destroy() throws MachineException {
+        try {
+            outputConsumer.close();
+        } catch (IOException ignored) {
+        }
```

Comment:
exception is logged in CompositeLineConsumer implementation
https://github.com/eclipse/che/blob/a8dfb6bf541940c636590119ef2fb45c1fa659c9/core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/CompositeLineConsumer.java#L36